readme.txt

```diff
@@ -246,14 +246,16 @@ Remember to always make a backup of your database and files before updating!
 * Fix - Restore erroneously removed Events query filters that would result in out-of-order results. [TEC-4770]
 * Fix - Correctly set `found_posts` and `max_num_pages` when redirecting a query to the custom tables. [BTRIA-1385]
 * Fix - Avoid resetting post data in some Dive theme and plugins. [BTRIA-1397]
+* Fix - Correctly set `found_posts` and `max_num_pages` when redirecting a query to the custom tables. [BTRIA-1385]
+* Fix - Avoid resetting post data in some Dive theme and plugins. [BTRIA-1397]
 * Tweak - Update the organizer website field to type URL. [TEC-4395]
 * Tweak - Update the venue website field to type URL. [TEC-4349]
 * Tweak - Add an event property for if the event is currently happening. [TBD]
 * Tweak - Create a filterable function `tec_events_get_today_button_label()` for the text on the "Today" button on calendar views. [TEC-4458]
 * Tweak - Add a filter for the link title and aria-label so they match the button text more closely. [TEC-4458]
 * Tweak - Added filters: `tec_events_today_button_label` and `tec_events_view_{view_slug}_today_button_label` to filter the output of the `tec_events_get_today_button_label` function
                          `tec_events_today_button_title` and `tec_events_{view_slug}_view_today_button_title` to filter the link title and aria-label. [TEC-4458]
-
+* Tweak - Speed up the Custom Tables v1 migration process if a browser window is open on the migration UI. [TBD]
```

Comment:
Do we have ticket for this?

Reply:
https://theeventscalendar.atlassian.net/browse/TEC-4517